api/v1alpha1/virtualmachineservice_types.go

```diff
@@ -85,6 +85,14 @@ type VirtualMachineServiceSpec struct {
 	// VirtualMachineService with the set of VirtualMachines that should back this VirtualMachineService.
 	Selector map[string]string `json:"selector"`
 
+	// Only applies to Service Type: LoadBalancer
+	// LoadBalancer will get created with the IP specified in this field.
+	// This feature depends on whether the underlying loadbalancer provider supports specifying
+	// the loadBalancerIP when a load balancer is created.
+	// This field will be ignored if the provider does not support the feature.
+	// +optional
+	LoadBalancerIP string `json:"loadBalancerIP,omitempty"`
+
 	// Just support cluster IP for now
```

Comment:
Can you remove this comment that would appear to not apply anymore? 

It is unfortunate that `clusterIp` instead of `clusterIP` was used in for the json name for ClusterIP.

Reply:
yeah, sure: https://github.com/vmware-tanzu/vm-operator-api/pull/41